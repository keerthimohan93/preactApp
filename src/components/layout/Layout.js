import { h } from 'preact';
import Header from '../header/Header';
import NavMenu from '../navMenu/NavMenu';
import style from './style';
const Layout = props => (
	<article class={style.layout}>
		<Header />
		<section>
			<NavMenu />
			{props.children}
		</section>
	</article>
);
export default Layout;
