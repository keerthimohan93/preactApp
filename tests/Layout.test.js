import Layout from '../src/components/layout/Layout';
// See: https://github.com/mzgoddard/preact-render-spy
import { shallow } from 'preact-render-spy';

describe('Initial Test of the Header', () => {
	test('it renders snapshot correctly', () => {
		const context = shallow(<Layout />);
		expect(context).toMatchSnapshot();
	});
});
