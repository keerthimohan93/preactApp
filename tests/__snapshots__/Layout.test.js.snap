// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Initial Test of the Header it renders snapshot correctly 1`] = `
RenderContext {
  "0": T {
    "attributes": undefined,
    "children": Array [],
    "key": undefined,
    "nodeName": [Function],
  },
  "length": 1,
}
`;
